models/public/index.md

```diff
@@ -113,6 +113,7 @@ SSD-based and provide reasonable accuracy/performance trade-offs.
 | Faster R-CNN with ResNet 50    | [TensorFlow\*](./faster_rcnn_resnet50_coco/faster_rcnn_resnet50_coco.md) | faster_rcnn_resnet50_coco | 57.203 | 29.162 |
 | Faster R-CNN with ResNet 101   | [TensorFlow\*](./faster_rcnn_resnet101_coco/faster_rcnn_resnet101_coco.md) |faster_rcnn_resnet101_coco | 112.052 | 48.128 |
 | MTCNN                     | Caffe\*: <br>[proposal](./mtcnn-p/mtcnn-p.md) <br>[refine](./mtcnn-r/mtcnn-r.md) <br> [output](./mtcnn-o/mtcnn-o.md) | <br>mtcnn-p <br>mtcnn-r <br>mtcnn-o| | |
+| Pelee                     | [Caffe\*](./pelee_coco/pelee_coco.md) | pelee_coco| | |
```

Comment:
Maybe it would be better to call it PeleeNet? The model is called PeleeNet in the article.

Reply:
no, it can mislead, PeleeNet is classification network used as backbone for this detection model (https://github.com/Robert-JunWang/PeleeNet). for detection net used Pelee as name.
 from article:
```
Model         Input    FLOPs Params   Data   mAP (%)
Pelee (ours) 304x304 1,210 M 5.43 M 07+12+COCO    76.4
```

Reply:
OK.